bin/cheritest/cheritest.h

```diff
@@ -165,7 +165,7 @@ void	signal_handler_clear(int sig);
 #define CHERITEST_CHECK_EQ_BOOL(a, b)	\
 	CHERITEST_CHECK_EQ(_Bool, "%d", a, b, __STRING(a), __STRING(b))
 #define CHERITEST_CHECK_EQ_INT(a, b)	\
-	CHERITEST_CHECK_EQ(int, "0x%lx", a, b, __STRING(a), __STRING(b))
+	CHERITEST_CHECK_EQ(int, "%d", a, b, __STRING(a), __STRING(b))
```

Comment:
Keep this consistent with long below, and ideally size_t too though I can see an argument for size_t being hex and the others being dec.